sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/implementation/TableUtils.java

```diff
@@ -612,16 +613,68 @@ public static String[] getKeysFromToken(String token) {
         return keys;
     }
 
-    public static <T> T callWithOptionalTimeout(Supplier<T> callable, ExecutorService threadPool, Duration timeout,
-        ClientLogger logger) {
+    public static Context skip409Logging(Context context) {
```

Comment:
Given that all these APIs are utility implementation, I don't think `Context` is the best way to handle this. We can just pass booleans around to handle whether to log.

Reply:
@alzimmermsft I'm not sure what you mean by utility implementation. Can you explain?

 My logic behind using `context` is that:

1. It's easy to change if we discover unexpected behaviors / reason to adjust logging or other things in the future.
2. It keeps the code flow consistent since `Context` is already in most method signatures.
3. I'm not a fan of passing a boolean through the handler that's false in almost every use case.

Is there a reason why `Context` _shouldn't_ be used?